.github/workflows/build.yml

```diff
@@ -270,6 +270,12 @@ jobs:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
+      -
+        name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4.3.0
+        with:
+          tag-sha: true # add git short SHA as Docker tag
```

Comment:
I didn't find this input field, only `tags`

Reply:
You're of course right, the documentation confused me a bit. I've now removed `tag-sha`.